lib/Stampie/MailerInterface.php

```diff
@@ -12,12 +12,12 @@
 interface MailerInterface
 {
     /**
-     * @param AdapterInterface $adapter
+     * @param HttpClient $adapter
      */
-    function setAdapter(AdapterInterface $adapter);
+    function setAdapter(HttpClient $adapter);
 
     /**
-     * @return AdapterInterface
+     * @return HttpClient
      */
     function getAdapter();
```

Comment:
I removed the `getAdapter`, but if you do not mind I suggest removing `setServerToken` would be a different PR.